index.js

```diff
@@ -0,0 +1,18 @@
+const fastify = require("fastify");
+const PORT = 3000;
+const app = fastify({ logger: true });
+
+app.get("/hello", async (request, reply) => {
+  return { message: "Hello World!" };
+});
+
+const start = async () => {
```

Comment:
```suggestion
async function start() {
```

Conceitualmente é a mesma coisa, mas acho que como convenção é legal utilizarmos `function` para funções e `const` para variáveis.